misc/src/main/java/com/ihren/misc/ObjectMapperFactory.java

```diff
@@ -0,0 +1,15 @@
+package com.ihren.misc;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class ObjectMapperFactory {
+    public static ObjectMapper create() {
```

Comment:
It is better to do it util class andput objectmapper to method because sometimes ObjectMapper can be already created